### ISSUE TITLE
Poor performance when fine-tuning on google colab

### ISSUE THREAD

**Issue by sarim-zafar** [explosion/spacy-transformers#113]
I followed the tutorial provided as-is on my binary text classification dataset. On the CPU it works as it should and I get good accuracy scores. but when I run the same code on Google colab I get very bad accuracy scores. I am however able to export the CPU pre-trained model online and get the same results on the test dataset.

**Comment by sarim-zafar** [comment]
@honnibal can you please look into this?

**Comment by svlandeg** [comment]
Could you provide more details on which exact scripts you were running on which data, and what the logs were?
